pkg/utils.go

```diff
@@ -5,14 +5,18 @@ import (
 	"strings"
 )
 
-func ptrStr(s string) *string {
-	return &s
+func expandEnv(str string) string {
+	str = strings.Replace(str, "$PWD", "{{ ALIASES_PWD }}", -1)
+	str = os.ExpandEnv(str)
+	str = strings.Replace(str,"{{ ALIASES_PWD }}", "${ALIASES_PWD:-$PWD}", -1)
```

Comment:
commandA (in Host) -> commandB (in Docker) is a special environment variable that holds PWD.